libs/hdf-converters/src/snyk-mapper.ts

```diff
@@ -37,132 +40,580 @@ function nistTag(identifiers: unknown[]): string[] {
+          status: 'loaded',
+          controls: [
+            {
+              path: isApi?.apiVersion === 1 ? 'issues' : 'vulnerabilities',
+              key: 'id',
+              tags: {
+                nist: {
+                  path: `${
+                    isApi?.apiVersion === 1 ? SNYK_API_ISSUE_DATA_PATH : ''
+                  }identifiers.CWE`,
+                  transformer: nistTag
+                },
+                cweid: {
+                  path: `${
+                    isApi?.apiVersion === 1 ? SNYK_API_ISSUE_DATA_PATH : ''
+                  }identifiers.CWE`,
+                  transformer: (input: unknown) => input || []
+                },
+                cveid: {
+                  path: `${
+                    isApi?.apiVersion === 1 ? SNYK_API_ISSUE_DATA_PATH : ''
+                  }identifiers.CVE`,
+                  transformer: (input: unknown) => input || []
+                },
+                ghsaid: {
+                  path: `${
+                    isApi?.apiVersion === 1 ? SNYK_API_ISSUE_DATA_PATH : ''
+                  }identifiers.GHSA`,
+                  transformer: (input: unknown) => input || []
+                },
+                exploitMaturity: {
+                  path: `${
+                    isApi?.apiVersion === 1 ? SNYK_API_ISSUE_DATA_PATH : ''
+                  }exploit${isApi?.apiVersion === 1 ? 'Maturity' : ''}`
+                },
+                CVSSv3: {
+                  path: `${
+                    isApi?.apiVersion === 1 ? SNYK_API_ISSUE_DATA_PATH : ''
+                  }identifiers.CVSSv3`
+                },
+                cvssScore: {
+                  path: `${
+                    isApi?.apiVersion === 1 ? SNYK_API_ISSUE_DATA_PATH : ''
+                  }identifiers.cvssScore`
+                }
+              },
+              refs: [],
+              source_location: {},
+              title: {
+                path: `${
+                  isApi?.apiVersion === 1 ? SNYK_API_ISSUE_DATA_PATH : ''
+                }title`
+              },
+              id: {path: 'id'},
+              desc: {
+                path: `${
+                  isApi?.apiVersion === 1 ? SNYK_API_ISSUE_DATA_PATH : ''
+                }description`,
+                transformer: (description: string): string => {
+                  return (
+                    description
+                      .split(/(?:^|\s)## /g)
+                      .find((section) => section.startsWith('Overview'))
+                      ?.substring('Overview'.length)
+                      ?.trim() || description
+                  );
+                }
+              },
+              descriptions: [
+                {
+                  label: 'fix',
+                  data: {
+                    path: `${
+                      isApi?.apiVersion === 1 ? SNYK_API_ISSUE_DATA_PATH : ''
+                    }description`,
+                    transformer: (description: string): string => {
```

Comment:
Add a guard saying if descriptions is undefined, not sure if this will ever happen but better safe than sorry